locale/zh-cn/index.md

```diff
@@ -18,8 +18,7 @@ labels:
   version-schedule-prompt-link-text: LTS 日程。
   newsletter: true
   newsletter-prefix: 订阅
-  newsletter-postfix: "，Node.js 官方的新闻周报。"
```

Comment:
This change seems to be unrelated.

Reply:
Yes, I just mean that removing the last pharagraph, with the image pasted above for you to see.

Reply:
Then maybe the PR title should be revised to cover that?

Reply:
OK. I'll update it